src/mcts/params.cc

```diff
@@ -160,6 +160,10 @@ const OptionId SearchParams::kMultiPvId{
     "multipv", "MultiPV",
     "Number of game play lines (principal variations) to show in UCI info "
     "output."};
+const OptionId SearchParams::kMultiPVDisaggregatedNodecountId{
+    "multipv-disaggregated-nodecount", "MultiPVDisaggregatedNodecount",
```

Comment:
I'm not sure that will fit UIs and not break layout in help.
Maybe something like `multipv-counters` ?